src/map/handler/Map.Keyboard.js

```diff
@@ -151,10 +151,15 @@ export var Keyboard = Handler.extend({
 					offset = toPoint(offset).multiplyBy(3);
 				}
 
-				map.panBy(offset);
-
 				if (map.options.maxBounds) {
-					map.panInsideBounds(map.options.maxBounds);
+					offset = map._limitOffset(toPoint(offset), map.options.maxBounds);
+				}
+
+				if (map.options.worldCopyJump) {
+					var newLatLng = map.wrapLatLng(map.unproject(map.project(map.getCenter()).add(offset)));
```

Comment:
This looks like something that could be moved to the `Map`'s `panBy()` method.

Reply:
I also thought about that but I do not completly understand how the original wrapping in Map.Drag works.

Reply:
I digged into it, it is not so easy. I created a new issue / task for this. #8625